src/test/java/org/apache/sysds/test/functions/privacy/FederatedL2SVMTest.java

```diff
@@ -314,7 +314,7 @@ private void federatedL2SVM(Types.ExecMode execMode, Map<String, PrivacyConstrai
 		if(rtplatform == Types.ExecMode.SPARK) {
 			DMLScript.USE_LOCAL_SPARK_CONFIG = true;
 		}
-		Process t1 = null, t2 = null;
+		Process[] ts = new Process[0];
```

Comment:
This has been changed to use threads since the use of processes failed in GitHub Actions.